platform/platform-logging/src/test/java/org/codice/ddf/logging/platform/LoggingServiceBeanTest.java

```
@@ -0,0 +1,258 @@
+        loggingServiceBean.init();
+        appendLogs(loggingServiceBean, mockPaxLoggingEvents);
+
+        // Perform Test
+        List<LogEvent> actualLogEvents = loggingServiceBean.retrieveLogEvents();
+
+        // Verify
+        assertThat(actualLogEvents,
+                contains(expectedLogEvents.toArray(new LogEvent[expectedLogEvents.size()])));
+    }
+
+    @Test
+    public void testResizeMaxLogEventsFrom500To1() {
+        // Setup
+        List<PaxLoggingEvent> mockPaxLoggingEvents = getMockPaxLoggingEventsTimestampOrderSmallestToLargest(3);
+        LogEvent expectedLogEventAfterResize = getExpectedLogEvent(mockPaxLoggingEvents.get(2));
+        LoggingServiceBean loggingServiceBean = new LoggingServiceBean();
+        loggingServiceBean.init();
+        appendLogs(loggingServiceBean, mockPaxLoggingEvents);
+
+        // Perform Test
+        loggingServiceBean.setMaxLogEvents(1);
+
+        // Verify
+        List<LogEvent> actualLogEvents = loggingServiceBean.retrieveLogEvents();
+        assertThat(actualLogEvents, contains(expectedLogEventAfterResize));
+    }
+
+    @Test
+    public void testResizeMaxLogEventsFrom500To600() {
+        // Setup
+        List<PaxLoggingEvent> mockPaxLoggingEvents = getMockPaxLoggingEventsTimestampOrderSmallestToLargest(3);
+        List<LogEvent> expectedLogEvents = getExpectedLogEvents(mockPaxLoggingEvents);
+        LoggingServiceBean loggingServiceBean = new LoggingServiceBean();
+        loggingServiceBean.init();
+        appendLogs(loggingServiceBean, mockPaxLoggingEvents);
+
+        // Perform Test
+        loggingServiceBean.setMaxLogEvents(600);
+
+        // Verify
+        List<LogEvent> actualLogEvents = loggingServiceBean.retrieveLogEvents();
+        assertThat(actualLogEvents,
+                contains(expectedLogEvents.toArray(new LogEvent[expectedLogEvents.size()])));
+    }
+
+    @Test
+    public void testSetMaxLogEventsTo0() {
```

Should we throw an exception if the max is set to 0?